gradle/verification-metadata.xml

```diff
@@ -1267,12 +1267,17 @@
       </component>
       <component group="org.gradle.kotlin" name="gradle-kotlin-dsl-conventions" version="0.8.0">
          <artifact name="gradle-kotlin-dsl-conventions-0.8.0.jar">
-            <sha256 value="b4182982109d85f9a623fa597932c3f09651467559bb99965c41d45159e89a68" origin="Artifact is not signed"/>
+            <sha256 value="b4182982109d85f9a623fa597932c3f09651467559bb99965c41d45159e89a68" origin="Artifact is not signed" reason="Artifact is not signed"/>
```

Comment:
Not to be fixed in this context but we should really change that release process to have them signed.